local-cli/bundle/bundleCommandLineArgs.js

```diff
@@ -18,8 +18,7 @@ module.exports = [
     default: 'ios',
   }, {
     command: '--transformer [string]',
-    description: 'Specify a custom transformer to be used (absolute path)',
-    default: require.resolve('../../packager/transformer'),
```

Comment:
Removing the default for command line arguments is pricely the point here (and the fact that #7899 added them again is the regression from #7961). We want the transformer to be specified as followed:

1. If specified on command line, use that
2. If specified by `rn-cli.config.js`, use that
3. Fallback to default configuration

For this fallback chain to work properly, there must not be a default value returned by the command line parser, otherwise we stop at (1) and never make it to (2).